frontend/user/vue-components/ApplicationLabel.vue

```diff
@@ -1,42 +1,44 @@
 <template>
   <ul v-if="currentApp !== null" class="nav navbar-nav">
-    <li class="dropdown">
+    <li class="dropdown notifications-menu">
       <a href="#" class="dropdown-toggle cust-padding" data-toggle="dropdown" aria-expanded="false">
         <img class="app-icon"
         :src="currentApp.appData.image.icon_128 | iconSrc">
 
         <span>{{ currentApp.appData.image | appName }}</span>
       </a>
 
-      <ul class="dropdown-menu" role="menu">
-        <li class="disabled">
-          <a href="#">Application settings</a>
-        </li>
-        <li role="separator" class="divider"></li>
+      <ul class="dropdown-menu">
+        <li class="header">Application settings</li>
```

Comment:
I would remove this entry here. It conflicts with HIG and brings in a problem we should not have to begin with.